apps/docs/blog/2025-02-27-Release-v0.11.0.mdx

```diff
@@ -0,0 +1,50 @@
+---
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+slug: v0.11.0
+title: Release 0.11.0
+authors:
+  - mellyeliu
+tags:
+  - release
+---
+
+# Release 0.11.0
+
+## Release Notes
+
+StyleX v0.11.0 is now available, with significant improvements in debugging, CLI caching, and documentation!
+
+## Debugging Improvements
+
+### Improved Runtime Debugging with SourceMap Data
+We've made some big improvements to the information available in debug mode. StyleX now includes **sourceMap information** to indicate the file and line of the `stylex.create` call responsible for styles on a given element. This information is contained in the `data-style-src` prop in the DOM and can be enabled by setting `debug: true` in the StyleX config.
+
+Additionally, we've introduced new flags for greater flexibility in fine-tuning debugging information:
```

Comment:
I don't think we should document these flags as we'll remove them once features are rolled out internally. For example, we don't document flags for react or react native